mk/PcapPlusPlus.mk.common

```diff
@@ -18,3 +18,12 @@ PCAPPP_BUILD_FLAGS :=
 ifdef PCAPPP_ENABLE_CPP_FEATURE_DETECTION
 	PCAPPP_BUILD_FLAGS += -DPCAPPP_CPP_FEATURE_DETECTION -std=c++11 
 endif
+
+ifndef CFLAGS
+CFLAGS := -O2 -g -Wall -c
+endif
+
+ifndef CXXFLAGS
+CXXFLAGS := -O2 -g -Wall -c
+endif

```

Comment:
I left the examples with their own build flags because I saw that they were slightly different (no optimization level instad of 2). If you'd like to have it there, no problem I will propagate it too.

Reply:
I don't think the optimization level is different on purpose 😃 

I think we can have the same optimization level across the lib and examples

Reply:
Good, I will apply those changes to the examples too.